### DISCUSSION TITLE
SSG Not working correctly

### DISCUSSION THREAD

**Discussion by serhatakti** [vercel/next.js#48388] [Help]
### Summary

Why are external requests that I use with getstaticprops also sent by the client after the site is created?
![image](https://user-images.githubusercontent.com/49954227/232055681-21cbcbac-01d7-4623-bc00-34e254808e8d.png)

**Comment by ciruz** [comment]
I think there is something else wrong.

[When does getStaticProps run](https://nextjs.org/docs/basic-features/data-fetching/get-static-props#when-does-getstaticprops-run)

Are you in development mode?

**Comment by nikkizol** [comment]
Have the same issue.
